Keycloak/terraform.tfvars

```diff
@@ -0,0 +1,35 @@
+# terraform.tfvars
+
+# AWS region to deploy the instance
+region = "us-east-2"
```

Comment:
Since most of the team is based in India, could we use `ap-south-1` for convenience?

Reply:
I’ve included instructions in the `README.md` that guide users to customize the `terraform.tfvars` file with their specific AWS configurations, including the region. Just edit the `terraform.tfvars` file with the preferred region and other relevant settings.